elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -21,8 +21,83 @@ $(function() {
 <div id="file-editor-{{uuid}}" class="mathjax_ignore">
     <input type="hidden" name="{{name}}">
     <div class="card">
-        <div class="card-header">{{file_name}}</div>
+        <div class="card-header d-flex justify-content-between align-items-center">
+             {{file_name}}
+             <button type="button" class="btn btn-sm btn-secondary settings-button" tabindex="-1">
+                 <div class="fa fa-cog"></div>
+             </button>
+         </div>
         <div class="editor"></div>
+        <div class="modal fade" tabindex="-1" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Editor Settings</h5>
```

Comment:
This is not an appropriate `id` for this modal. You should incorporate the `uuid` in case there are multiple file editors on the page.

Reply:
By uuid do you mean the uuid passed in when creating editor instance?

Reply:
Actually I was thinking we should add `id` to line 32 instead. Shouldn't it be on the entire modal itself instead of on modal-title?

Reply:
No, the ID should be on the title, and the modal itself should reference that ID via `aria-labelledby="..."`. See https://getbootstrap.com/docs/4.6/components/modal/#accessibility for more details.